### DISCUSSION TITLE
Ask M109 When will the push be deployed?

### DISCUSSION THREAD

**Discussion by whxuezhe** [Alex313031/Thorium-Win-AVX2#48] [General]
Chrome Canary has been 109.0 for two days, how long has Thorium been planning this time, it has been 24 days since the first version of M108 was released!

**Comment by gz83** [comment]
> Chrome Canary has been 109.0 for two days, how long has Thorium been planning this time, it has been 24 days since the first version of M108 was released!

@whxuezhe Hello, please be patient and wait for the release of the new version
